README.md

```diff
@@ -176,6 +176,19 @@ Pipeline is defined in `.buildkite/pipeline.yml`
 * Tests: Uses the official [python:3.6](https://hub.docker.com/_/python/) image.
 * Style: Uses [black](https://hub.docker.com/r/stellargraph/black/) from the `stellargraph` docker hub organisation.
 
+## Citing
+If you use any part of this library in your research, please cite it using the following BibTex entry
+```latex
+@misc{stellargraph,
+  author = {Data61},
```

Comment:
We need to use "CSIRO's Data61". We should also mention this somewhere else e.g. 
"StellarGraph is proudly designed, developed and supported by [CSIRO's Data61](https://data61.csiro.au)". Perhaps in the references? Or at the top of the citing section?

Reply:
Good idea, although I might remove "proudly"?